node/src/actors/chain_manager/mining.rs

```diff
@@ -836,6 +953,34 @@ fn build_block(
     (block_header, txns)
 }
 
+fn build_superblock(
```

Comment:
Should this be in `impl SuperBlock`

Reply:
now it is similar to build block, a private function in mining. Moreover, moving that function to data_structures creates problems with cyclic dependencies that maybe could solved in another issue